nemo/collections/tts/parts/preprocessing/features.py

```diff
@@ -333,3 +378,10 @@ def load(self, manifest_entry: dict, audio_dir: Path, feature_dir: Path) -> Dict
             feature_dir=feature_dir,
         )
         return feature_dict
+
+    def collate_fn(self, train_batch: List[Dict[str, Tensor]]) -> Dict[str, Tensor]:
+        feature_dict = {}
+        _collate_feature(feature_dict=feature_dict, feature_name=self.pitch_name, train_batch=train_batch)
+        _collate_feature(feature_dict=feature_dict, feature_name=self.voiced_mask_name, train_batch=train_batch)
+        _collate_feature(feature_dict=feature_dict, feature_name=self.voiced_prob_name, train_batch=train_batch)
```

Comment:
We're not guaranteed to have (or need) each of these features all the time. Should there be checks before each collate to make sure it's necessary?